docs/info/version_policy.md

```diff
@@ -45,3 +34,25 @@ v4| July 2018 | January 2021
 v3| September 2017 | September 2020
 v2| November 2016 | December 2019
 v1| December 2015 | July 2018
+
+:::tip
+The current version of the Platform is visible in the Console info section.
+:::
+
+## Feature Preview and Beta
+
+New Mia-Platform version may include features that are not yet considered stable; such features are highlighed with a special
+tag denoting the current feature state.
+
+:::info
+Preview and beta feature are usually disabled by default and can be enabled using the Feature Preview pages (depending whether the feature has impact on [specific users](/docs/development_suite/user-settings/feature-preview)
+or whole [Projects](/docs/console/project-configuration/project-settings#feature-preview)).
+:::
+
+The currently used feature states are the following:
+
+- **experimenta**: used for feature that are under heavy development;
+- **preview**: generally representing the first iteration for a new feature;
+it means that the feature is available for use but it may not be completed yet or could be subject to undocumented breaking changes;
+- **beta**: the feature implementation is completed and there will not be any undocumented breaking changes. However, for different reasons
+(e.g. lack of proper validatiom on certain use-cases), it may be yet considered unsuitable for use in production, hence it must be used carefully.
```

Comment:
```suggestion
(e.g. lack of proper validation on certain use-cases), it may be yet considered unsuitable for use in production, hence it must be used carefully.
```